### ISSUE TITLE
How can I resize the model from the left side? I found that I can only resize the model from the right side using your default code.

### ISSUE THREAD

**Issue by CatherineJieWang** [wwan5803/react-modal-resizable-draggable#21]
How can I resize the model from the left side? I found that I can only resize the model from the right side using your default code.

_Originally posted by @CatherineJieWang in https://github.com/wwan5803/react-modal-resizable-draggable/issues/4#issuecomment-509070668_

**Comment by wwan5803** [comment]
Yep, you can only resize the model from the right side. Isn't that enough? Unless people from some countries read from left to right. I might add this feature later.
